src/inmanta/protocol/methods_v2.py

```diff
@@ -1328,3 +1328,52 @@ def get_environment_metrics(
     :param end_interval: The end of the time window for which the metrics should be returned.
     :param nb_datapoints: The amount of datapoint that will be returned within the given time interval for each metric.
     """
+
+
+@typedmethod(path="/login", operation="POST", client_types=[ClientType.api], enforce_auth=False, api_version=2)
+def login(username: str, password: str) -> ReturnValue[model.LoginReturn]:
```

Comment:
Why `ReturnValue`? Isn't that the old approach? I could be mistaken.

Reply:
I have no idea. What would be the new approach? @arnaudsjs do you know about this?

Reply:
Sorry, I meant to withdraw this comment. The new approach is just returning the object, but if you want to explicitly include a bearer token (as you do) that doesn't suffice.